frontend/package.json

```diff
@@ -24,12 +24,12 @@
     "@uniswap/token-lists": "^1.0.0-beta.19",
     "@unstoppabledomains/resolution": "8.3.1",
     "@vue/composition-api": "^0.6.4",
-    "@web3-onboard/coinbase": "^2.1.1",
-    "@web3-onboard/core": "^2.8.1",
-    "@web3-onboard/injected-wallets": "^2.2.0",
-    "@web3-onboard/ledger": "^2.2.1",
-    "@web3-onboard/trezor": "^2.2.1",
-    "@web3-onboard/walletconnect": "^2.1.1",
+    "@web3-onboard/coinbase": "^2.1.3",
```

Comment:
_(arbitrary comment location to start a thread)_

Seems that when I cancel a withdraw to a token after the warning, the withdraw form is stuck in this loading state

<img width="602" alt="image" src="https://user-images.githubusercontent.com/17163988/196263129-8309f859-9e64-4d42-beb6-30b825564086.png">

Reply:
This should be resolved